assets/javascripts/omise-embedded-card.js

```diff
@@ -66,4 +66,20 @@ function showOmiseEmbeddedCardForm({
     onCreateTokenSuccess: onSuccess ?? noop,
     onError: onError ?? noop
   });
+
+  preventAutoScrollToSecureForm()
+}
+
+// Link: https://stackoverflow.com/questions/6596668/iframe-on-the-page-bottom-avoid-automatic-scroll-of-the-page
+function preventAutoScrollToSecureForm() {
```

Comment:
* Will this need to be propagated across all users of the secure card embedded form? e.g. Magento
* Is there any room to move the responsibility for this to pay.js itself?

Reply:
- Will this need to be propagated across all users of the secure card embedded form? e.g. Magento
   - I haven't tested it in Magento but it's safe to assume yes.
- Is there any room to move the responsibility for this to pay.js itself?
   - It's OmiseJS creating the iframe, not PayJS but I did the same logic there first. It didn't work.
   
  The reasons on why it didn't work could be that
  - we are attaching the iframe to an element of the parent page (WooCommerce in this case).
  - the iframe's code originates from a different domain than the parent page.

These restrictions might have prevented it from accessing and manipulating elements outside the iframe's boundaries.

Reply:
What I mean is, can the form element of pay.js that is stealing focus be made to not do so?